tests/test_actions.py

```diff
@@ -971,3 +972,94 @@ def perform_as(self, the_actor: Actor) -> None:
         # kinked microphone will flush the logs back up into the previous
         # kink, which will then be cleared by the outer Silently.
         assert [r.msg for r in caplog.records] == []
+
+
+class TestTryTo:
+    def test_can_be_instantiated(self) -> None:
+        t1 = TryTo()
```

Comment:
Should we say anything if someone doesn't try to do anything? We've got our cheeky message in `SeeAnyOf` and `SeeAllOf`.

Reply:
I had thought this was more akin to `actor.attempts_to()`